Cargo.toml

```diff
@@ -23,16 +23,21 @@ log = "0.4"
 rand = "0.4"
 serde = "1.0"
 serde_derive = "1.0"
+sha2 = "^0.7"
+hkdf = "^0.5"
+ring = "^0.12"
```

Comment:
Bump this to `0.13` (also the `^` is redundant... Cargo is semver-by-default)

Reply:
thanks, done